test/v2/models/response/retrieveBalanceAndTransactions/DocumentDetailsSpec.scala

```diff
@@ -58,16 +51,16 @@ class DocumentDetailsSpec extends UnitSpec with JsonErrorValidators {
 
       "convert 'i' to true" in {
```

Comment:
I think we need to replace 'i' with 'K' here (line 52)

Reply:
Updated.